samples/snippets/fsharp/lang-ref-1/snippet4002.fs

```diff
@@ -1,6 +1,6 @@
-let rec Even x =
-   if x = 0 then true
-   else Odd (x - 1)
-and Odd x =
-   if x = 1 then true
-   else Even (x - 1)
\ No newline at end of file
+let rec Even x = 
+    if x = 0 then true 
+    else Odd (x-1) 
+and Odd x = 
+    if x = 0 then false 
+    else Event (x-1)
```

Comment:
`Event` should be `Even` - looks good otherwise.  Thanks for catching the `true`/`false` bug!